src/translate.rs

```diff
@@ -213,6 +214,19 @@ impl ToGlib for Option<char> {
     }
 }
 
+impl ToGlib for Ordering {
+    type GlibType = i32;
+
+    #[inline]
+    fn to_glib(&self) -> i32 {
+        match *self {
+            Ordering::Less => -1,
+            Ordering::Equal => 0,
+            _ => 1,
```

Comment:
Um, could you spell out the last variant please? They deserve equal treatment ;)